service/controller/resource/monitoring/pvcresizingresource/create.go

```diff
@@ -30,21 +30,16 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		}
 
 		for _, pvc := range pvcList {
-			fmt.Println("PVC Name", pvc.GetName())
 			currentPVCSize := resource.MustParse(pvc.Spec.Resources.Requests.Storage().String())
 			desiredPVCSize := cluster.GetAnnotations()[key.PrometheusVolumeSizeAnnotation]
```

Comment:
Looking really good but can you add some logs in this resource?

Reply:
which one?

Reply:
A bit of debug mostly :) something like resizing, changing pvc and so on

Reply:
I'd go with the following on every code branch

// resizinv PVC xxx
resizing code...
// Resizing succeeded/failed

Reply:
can you take a look

Reply:
Nice